<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>needs</class>
 <widget class="QWidget" name="needs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>装备能力需求</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Adobe Gothic Std B</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>各阶段装备能力需求</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>初步打通阶段</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>扩大抢通阶段</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>清理加固阶段</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="2" column="1">
    <widget class="QListWidget" name="listWidget_2"/>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QListWidget" name="listWidget_3"/>
   </item>
   <item row="3" column="0">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="comboBox_2"/>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QComboBox" name="comboBox_3"/>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pb_add_1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>46</height>
      </size>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pb_add_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>46</height>
      </size>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QPushButton" name="pb_add_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>46</height>
      </size>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="5">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>48</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pb_back">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>46</height>
      </size>
     </property>
     <property name="text">
      <string>上一步</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="pb_next">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>46</height>
      </size>
     </property>
     <property name="text">
      <string>下一步</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>双击已有项可删除</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
